renderlib/graphics/glsl/GLThickLines.cpp

```diff
@@ -1,12 +1,10 @@
 #include "GLThickLines.h"
 
+#include "glm.h"
```

Comment:
these changes are just a cleanup thing, where my "glm.h" itself already includes several of the "<glm/glm....hpp>" includes.